frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -534,7 +534,7 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
     },
     flags: {
-      disallowed: [OCS_INDEPENDENT_FLAG, MCG_STANDALONE, FEATURES.ADD_CAPACITY],
+      disallowed: [OCS_INDEPENDENT_FLAG, MCG_STANDALONE],
```

Comment:
1) No need to backport to OCP 4.10, this flag is added to this extension in `release-4.10` also but CSV do not have this flag in ODF 4.10
2) If we want to disable from CSV, ptal: https://github.com/red-hat-storage/ocs-operator/pull/1681

Let me know which PR to close and which to use.

Reply:
Let's close this PR and remove it from the operator.

Reply:
let's remove it from both no point in keeping this dummy flag here.

Reply:
When we will have dynamic extension for this we can simply add that to `odf-console` and enable the flag from `ocs-operator` and we will not have to come to `console` and add this disallowed flag again.
So IMO we can close this PR ??